openrtb_ext/request.go

```diff
@@ -32,6 +32,7 @@ type ExtRequestPrebid struct {
 	Channel              *ExtRequestPrebidChannel  `json:"channel,omitempty"`
 	Data                 *ExtRequestPrebidData     `json:"data,omitempty"`
 	Debug                bool                      `json:"debug,omitempty"`
+	Integration          string                    `json:"integration,omitempty"`
```

Comment:
If a request contains `.ext.prebid`, `integration` will always be set given that this is a string and not a pointer to a string. In that case, if `integration` is not specified it will be an empty string. Is the plan to just check for an empty string when determining whether to apply the default integration value? Is there a use case where a publisher does not want to set an integration type, in which case they would explicitly set it to empty string? If so, then a pointer to a string would be needed as we would need to detect presence to differentiate between not set and set to empty string.

Reply:
This is a good question to bring up in tomorrow's standup, but the current implementation I have is if integration is blank then we apply the default integration value. 

However, the answer for how a publisher could omit an integration type in general is unclear. They could just set their default integration value to be blank, but I don't know if that approaches is what we want.

Reply:
@mansinahar thoughts on this? Do we need to confirm this behavior with Bret? I did a little digging and it appears this implementation matches what's in PBS-Java and the edge case I described above is unaccounted for and perhaps not valid. In `Ortb2RequestFactory.java`, the `StringUtils.isBlank(integration)` call checks that the requeset integration is null or empty string before applying the default integration.

Reply:
@bsardo Yeah I think this should be fine. I don't see a specific use case for a publisher to be able to not specify any integration at all being mentioned in the issue description #1428. And as you mentioned it matches with the Java implementation as well.